firewalld/config.json.template

```diff
@@ -0,0 +1,388 @@
+					"CAP_SYS_MODULE",
+					"CAP_SYS_RAWIO",
+					"CAP_SYS_CHROOT",
+					"CAP_SYS_PTRACE",
+					"CAP_SYS_PACCT",
+					"CAP_SYS_ADMIN",
+					"CAP_SYS_BOOT",
+					"CAP_SYS_NICE",
+					"CAP_SYS_RESOURCE",
+					"CAP_SYS_TIME",
+					"CAP_SYS_TTY_CONFIG",
+					"CAP_MKNOD",
+					"CAP_LEASE",
+					"CAP_AUDIT_WRITE",
+					"CAP_AUDIT_CONTROL",
+					"CAP_SETFCAP",
+					"CAP_DAC_OVERRIDE",
+					"CAP_MAC_OVERRIDE",
+					"CAP_DAC_READ_SEARCH",
+					"CAP_MAC_ADMIN",
+					"CAP_SYSLOG",
+					"CAP_WAKE_ALARM",
+					"CAP_BLOCK_SUSPEND",
+					"CAP_AUDIT_READ"
+				],
+				"effective": [
+					"CAP_CHOWN",
+					"CAP_FOWNER",
+					"CAP_FSETID",
+					"CAP_KILL",
+					"CAP_SETGID",
+					"CAP_SETUID",
+					"CAP_SETPCAP",
+					"CAP_LINUX_IMMUTABLE",
+					"CAP_NET_BIND_SERVICE",
+					"CAP_NET_BROADCAST",
+					"CAP_NET_ADMIN",
+					"CAP_NET_RAW",
+					"CAP_IPC_LOCK",
+					"CAP_IPC_OWNER",
+					"CAP_SYS_MODULE",
+					"CAP_SYS_RAWIO",
+					"CAP_SYS_CHROOT",
+					"CAP_SYS_PTRACE",
+					"CAP_SYS_PACCT",
+					"CAP_SYS_ADMIN",
+					"CAP_SYS_BOOT",
+					"CAP_SYS_NICE",
+					"CAP_SYS_RESOURCE",
+					"CAP_SYS_TIME",
+					"CAP_SYS_TTY_CONFIG",
+					"CAP_MKNOD",
+					"CAP_LEASE",
+					"CAP_AUDIT_WRITE",
+					"CAP_AUDIT_CONTROL",
+					"CAP_SETFCAP",
+					"CAP_DAC_OVERRIDE",
+					"CAP_MAC_OVERRIDE",
+					"CAP_DAC_READ_SEARCH",
+					"CAP_MAC_ADMIN",
+					"CAP_SYSLOG",
+					"CAP_WAKE_ALARM",
+					"CAP_BLOCK_SUSPEND",
+					"CAP_AUDIT_READ"
+				],
+				"ambient": [
+					"CAP_CHOWN",
+					"CAP_FOWNER",
+					"CAP_FSETID",
+					"CAP_KILL",
+					"CAP_SETGID",
+					"CAP_SETUID",
+					"CAP_SETPCAP",
+					"CAP_LINUX_IMMUTABLE",
+					"CAP_NET_BIND_SERVICE",
+					"CAP_NET_BROADCAST",
+					"CAP_NET_ADMIN",
+					"CAP_NET_RAW",
+					"CAP_IPC_LOCK",
+					"CAP_IPC_OWNER",
+					"CAP_SYS_MODULE",
+					"CAP_SYS_RAWIO",
+					"CAP_SYS_CHROOT",
+					"CAP_SYS_PTRACE",
+					"CAP_SYS_PACCT",
+					"CAP_SYS_ADMIN",
+					"CAP_SYS_BOOT",
+					"CAP_SYS_NICE",
+					"CAP_SYS_RESOURCE",
+					"CAP_SYS_TIME",
+					"CAP_SYS_TTY_CONFIG",
+					"CAP_MKNOD",
+					"CAP_LEASE",
+					"CAP_AUDIT_WRITE",
+					"CAP_AUDIT_CONTROL",
+					"CAP_SETFCAP",
+					"CAP_DAC_OVERRIDE",
+					"CAP_MAC_OVERRIDE",
+					"CAP_DAC_READ_SEARCH",
+					"CAP_MAC_ADMIN",
+					"CAP_SYSLOG",
+					"CAP_WAKE_ALARM",
+					"CAP_BLOCK_SUSPEND",
+					"CAP_AUDIT_READ"
+				]
+			},
+			"noNewPrivileges": false
+		},
+		"root": {
+			"path": "rootfs",
+			"readonly": true
+		},
+                "hooks": {},
+		"linux": {
+			"resources": {
+				"devices": [
+					{
+						"allow": true,
+						"access": "rwm"
+					}
+				]
+			},
+			"namespaces": [
+				{
+					"type": "mount"
+				}
+			]
+		},
+		"mounts": [
+		{
+			"destination": "/proc",
+			"type": "proc",
+			"source": "proc"
+		},
+		{
+			"type": "bind",
+			"source": "/dev",
+			"destination": "/dev",
+			"options": [
+				"rbind",
+				"rw",
+				"mode=755"
+			]
+		},
+		{
+			"type": "bind",
```

Comment:
we don't need to bind mount `/dev/log` as we are already bind mounting `/dev`

Reply:
ah, yup, I copied this from net-snmp config.json.template file :p, will drop it then :).